infra/aws/terraform/prow-build-cluster/hack/flux-update.bash

```diff
@@ -76,14 +76,18 @@ flux create source helm kubecost \
     --interval=${sync_interval} \
     --export >> ${resources_dir}/flux-system/flux-source-helm-kubecost-chart.yaml
 
-# The helm values are different for canary and prod clusters
+# Different helm values are provided canary and prod clusters via ConfigMap
+kubectl create configmap kubecost-helm-values \
```

Comment:
`kubectl create` doesn't handle updates, if the file gets changed, this command would fail with something like:

```
error: failed to create configmap: configmaps "my-config" already exists
```

I recommend chaining `kubectl create` and `kubectl apply`, something like `kubectl create --dry-run=client -o yaml | kubectl apply -f -`